code/modules/mob/new_player/new_player.dm

```diff
@@ -86,8 +86,7 @@
 
 	..()
 
-	statpanel("Lobby")
```

Comment:
This statpane is still used by admins and many other things, including the starting delay and list of ready players (for admins).

I wouldn't be opposed to adding the time till start to status, but leave lobby as it is.

Reply:
The delay and list of ready players just gets transferred to the Status tab, I tested it with admin rights

Reply:
Old status tab
![20220514-001038-dreamseeker](https://user-images.githubusercontent.com/7831163/168403230-2b55edfc-10df-488e-8797-6bd81b3a28a6.png)

Old lobby tab
![20220514-001030-dreamseeker](https://user-images.githubusercontent.com/7831163/168403215-d8e9eec1-8cd0-4dec-a264-eca5395a089a.png)

New status tab
![20220514-001355-dreamseeker](https://user-images.githubusercontent.com/7831163/168403244-bddf954c-4495-44d8-b3bf-c3d59ac6b20a.png)

(all screenshots taken as admin)

All the info is there.

Reply:
This lumps all the ready players into status. That's a huge no from me.

The status tab lags some people enough to the point where we have a reduced mode. If we have a list of 100 people in it, status will become laggy to the point where the MC tab is preferable to it, and I don't say that lightly.

Reply:
Alright so keeping the Lobby tab for admin only stuff is that fine? @AffectedArc07

Reply:
> Alright so keeping the Lobby tab for admin only stuff is that fine? @AffectedArc07

Just add `Time till start` as a display in `Status` and leave lobby exactly how it is.